examples/flutter_gallery/lib/demo/cupertino/cupertino_navigation_demo.dart

```diff
@@ -240,19 +261,21 @@ class Tab1RowItem extends StatelessWidget {
               ),
               CupertinoButton(
                 padding: EdgeInsets.zero,
-                child: const Icon(CupertinoIcons.plus_circled,
+                child: const Icon(
+                  CupertinoIcons.plus_circled,
                   color: CupertinoColors.activeBlue,
                   semanticLabel: 'Add',
                 ),
-                onPressed: () { },
+                onPressed: () {},
```

Comment:
fwiw, we changed the style guide on this to be less opinionated, but personally i still feel like `{}` looks like an empty map while `{ }` looks like an empty code block. Not sure why I feel that way!